internal/venice-common/src/main/java/com/linkedin/venice/meta/VersionImpl.java

```diff
@@ -298,6 +298,14 @@ public void setUseVersionLevelIncrementalPushEnabled(boolean versionLevelIncreme
     this.storeVersion.useVersionLevelIncrementalPushEnabled = versionLevelIncrementalPushEnabled;
   }
 
+  @Override
+  public boolean isHybrid() {
+    HybridStoreConfig hybridStoreConfig = getHybridStoreConfig();
+    return hybridStoreConfig != null
+        && (hybridStoreConfig.getRewindTimeInSeconds() >= 0 || hybridStoreConfig.getOffsetLagThresholdToGoOnline() >= 0
+            || hybridStoreConfig.getProducerTimestampLagThresholdToGoOnlineInSeconds() >= 0);
+  }
```

Comment:
If `VeniceHelixAdmin.isHybrid `and here share the same logic, could we abstract it out and move the logic to `Utils` class to avoid duplicating codes.